pkg/controllers/dynakube/activegate/internal/statefulset/builder/modifiers/authtoken.go

```diff
@@ -25,7 +25,7 @@ type AuthTokenModifier struct {
 }
 
 func (mod AuthTokenModifier) Enabled() bool {
-	return mod.dynakube.UseActiveGateAuthToken()
+	return mod.dynakube.NeedsActiveGate()
```

Comment:
```suggestion
	return true // TODO: Investigate moving this package to the default statefulset
```
in this context (an internal activegate package), I think using `NeedsActiveGate` is a bit weird 😅